traversal/producer/VertexProducer.java

```diff
@@ -45,7 +45,12 @@ public VertexProducer(GraphManager graphMgr, ProcedureVertex<?, ?> vertex, Trave
 
     @Override
     public void produce(Sink<VertexMap> sink, int count) {
-        if (future == null) future = runAsync(consume(count, sink), forkJoinPool());
+        if (future == null) {
+            future = runAsync(consume(count, sink), forkJoinPool()).exceptionally(e -> {
```

Comment:
Whenever we have to spawn an async job, we need not forget to add exception handling. If we forget, the consumer may stall and stalling is hard to debug.

I am open to any ideas on making it less error-prone.